src/main/java/com/synopsys/integration/detect/tool/detector/DetectorRuleFactory.java

```diff
@@ -99,6 +100,8 @@ public DetectorRuleSet createRules(DetectDetectableFactory detectableFactory) {
         }).allEntryPointsFallbackToNext();
 
         rules.addDetector(DetectorType.CONAN, detector -> {
+            detector.entryPoint(Conan2CliDetectable.class)
```

Comment:
In the case of Conan 2 this new detectable provides the most detailed and accurate results.
For Conan 1 ConanLockfileDetectable is still preferred over the CLI one.